getVideo.js

```diff
@@ -21,17 +21,11 @@ export async function main(event, context, callback) {
   try {
     const result = await Video.findAll({ where: { "id": id } });
     statusCode = 200;
-    let body = JSON.stringify(result[0]);
-
-    if (result.length === 0) {
-      statusCode = 204;
```

Comment:
You can leave this here, just change the status code to 404. The frontend will check the status code and display the appropriate page.